index.js

```diff
@@ -3,8 +3,73 @@
 const Scm = require('screwdriver-scm-base');
 const hoek = require('hoek');
 const url = require('url');
+const request = require('request');
+const schema = require('screwdriver-data-schema');
+const API_URL = 'https://api.bitbucket.org/2.0/repositories/';
```

Comment:
This will be done after we have the `setup()` right? Or you want me to add that now too?